src/button-actions.js

```diff
@@ -0,0 +1,35 @@
+
+import BackgorundImage from './images/Backgorund4.jpg';
+
+const navButton = () => {
+  const menu = document.querySelectorAll('.menu');
+  function onclickAction(item, position, bg) {
+    item.onclick = () => {
+      const content = document.querySelectorAll('.contentInfo');
+      content[position].style.display = 'grid';
+      const body = document.querySelector('body');
+      body.style.backgroundImage = `url(${bg})`;
+
+      item.style.backgroundColor = '#ff9035';
+      item.style.color = 'rgb(53, 7, 1)';
+      for (let index = 0; index < content.length; index += 1) {
+        if (index !== position) {
+          content[index].style.display = 'none';
+        }
+        if (index !== position - 1) {
+          menu[index].style.color = '#ff9035';
+          menu[index].style.backgroundColor = '';
+        }
+      }
+    };
```

Comment:
Use of arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) This applies to all the modules where you relied on the `function` keyword for your functions.